### PR TITLE
Set default scala version to 2.11.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

**Comment by markschaake** [comment]
LGTM

File: build.sbt [hunk 1]
```diff
@@ -17,6 +17,7 @@ scalacOptions := Seq(
   "-language:_",
   "-Xlog-reflective-calls")
 
+// SBT requires 2.10 for now (1/15/15).
```
Comment (jkinkead) [inline build.sbt]: So, this sets the default version within the core dependencies plugin, but not for the plugins library itself? That makes sense.

**Comment by jkinkead** [comment]
Non-sarcastic question! If the answer is "yes", LGTM! If it's not, I'd be confused, and would like to understand better what's going on.

File: build.sbt (hunk 1 above)
Comment (schmmd) [inline build.sbt]: Yes.  The plugins library needs to compile for 2.10.  I get confused by this every time--I keep trying to bump it up to 2.11 but that's impossible.

Comment (jkinkead) [inline build.sbt]: Sweet. Merging.

**Pull request merged**